test/generate_test_advanced.c

```diff
@@ -0,0 +1,1117 @@
+/*
+ * Copyright (c) 2015, Maxime SCHMITT
+ * All rights reserved.
+ *
```

Comment:
Could you consider to change the license to LGPL 2.1+ to be in line with the remainder of CLooG ?